src/Raven.Client/Util/AtomicDictionary.cs

```diff
@@ -163,4 +166,30 @@ public IDisposable TryWithAllLocks()
             return _globalLocker.TryEnterWriteLock(TimeSpan.FromSeconds(3));
         }
     }
-}
\ No newline at end of file
+
+    internal class AtomicDictionaryComparer : IEqualityComparer<RequestDestination>
```

Comment:
create internal comparer for DatabaseChangesOptions

Reply:
done